test/jdk/java/jextract/SmokeTest.java

```diff
@@ -52,6 +58,28 @@ public void testParser() {
         checkConstant(d, "ZERO", intType, 0L);
     }
 
+    @Test
+    public void test8238712() {
```

Comment:
At some point (e.g. separate changeset) it would be nice to split the tests from the framework here. E.g. SmokeTest has accumulated a lot of useful stuff which allow us to write API tests easily. But, currently, we are putting the tests together with the framework. We should make the framework a superclass, and then have a bunch of separate test subclasses - one per header.